qiskit_ibm_runtime/base_runtime_job.py

```diff
@@ -378,9 +377,7 @@ def usage_estimation(self) -> Dict[str, Any]:
 
     @property
     def instance(self) -> Optional[str]:
-        """For ibm_quantum channel jobs, return the instance where the job was run.
-        For ibm_cloud and ibm_quantum_platform, `None` is returned.
-        """
+        """For ibm_cloud and ibm_quantum_platform, `None` is returned."""
```

Comment:
This line was changed in #2296 (let's make sure it's not overwritten when merging)